### ISSUE TITLE
Build error with Angular Universal (SSR)

### ISSUE THREAD

**Issue by pschuster** [TypeCtrl/ngx-emoji-mart#263]
Hi,

I really like this component and added it to our project to add some emoji support to our community systems (like forum and chats). However, our project is also using Angular Universal. Just adding the `PickerModule` to the `AppModule` (which is then imported in the `ServerModule` gives me this error when running `npm run build:ssr`. Our stack is pretty standard with Angular 8.2.7 and ngx-emoji-mart with version 1.02.

Thank you very much for this awesome component and I hope this can be resolved quickly.

Cheers,
Phillip

This is the error:
```
ERROR in ./node_modules/@ctrl/ngx-emoji-mart/ngx-emoji/fesm5/ctrl-ngx-emoji-mart-ngx-emoji.js
Module build failed (from ./node_modules/@angular-devkit/build-optimizer/src/build-optimizer/webpack-loader.js):
TypeError: Cannot read property 'kind' of undefined
    at isAngularDecoratorMetadataExpression (/Users/pschuster/Documents/Websites/fourplayers/node_modules/@angular-devkit/build-optimizer/src/transforms/scrub-file.js:230:35)
    at checkNodeForDecorators (/Users/pschuster/Documents/Websites/fourplayers/node_modules/@angular-devkit/build-optimizer/src/transforms/scrub-file.js:53:21)
    at visitNodes (/Users/pschuster/Documents/Websites/fourplayers/node_modules/typescript/lib/typescript.js:16631:30)
    at Object.forEachChild (/Users/pschuster/Documents/Websites/fourplayers/node_modules/typescript/lib/typescript.js:16857:24)
    at checkNodeForDecorators (/Users/pschuster/Documents/Websites/fourplayers/node_modules/@angular-devkit/build-optimizer/src/transforms/scrub-file.js:44:31)
    at visitNode (/Users/pschuster/Documents/Websites/fourplayers/node_modules/typescript/lib/typescript.js:16622:24)
    at Object.forEachChild (/Users/pschuster/Documents/Websites/fourplayers/node_modules/typescript/lib/typescript.js:16752:21)
    at checkNodeForDecorators (/Users/pschuster/Documents/Websites/fourplayers/node_modules/@angular-devkit/build-optimizer/src/transforms/scrub-file.js:44:31)
    at visitNode (/Users/pschuster/Documents/Websites/fourplayers/node_modules/typescript/lib/typescript.js:16622:24)
    at Object.forEachChild (/Users/pschuster/Documents/Websites/fourplayers/node_modules/typescript/lib/typescript.js:16809:24)
    at checkNodeForDecorators (/Users/pschuster/Documents/Websites/fourplayers/node_modules/@angular-devkit/build-optimizer/src/transforms/scrub-file.js:44:31)
    at visitNode (/Users/pschuster/Documents/Websites/fourplayers/node_modules/typescript/lib/typescript.js:16622:24)
    at Object.forEachChild (/Users/pschuster/Documents/Websites/fourplayers/node_modules/typescript/lib/typescript.js:16820:24)
    at checkNodeForDecorators (/Users/pschuster/Documents/Websites/fourplayers/node_modules/@angular-devkit/build-optimizer/src/transforms/scrub-file.js:44:31)
    at visitNode (/Users/pschuster/Documents/Websites/fourplayers/node_modules/typescript/lib/typescript.js:16622:24)
    at Object.forEachChild (/Users/pschuster/Documents/Websites/fourplayers/node_modules/typescript/lib/typescript.js:16716:21)
    at checkNodeForDecorators (/Users/pschuster/Documents/Websites/fourplayers/node_modules/@angular-devkit/build-optimizer/src/transforms/scrub-file.js:44:31)
    at visitNodes (/Users/pschuster/Documents/Websites/fourplayers/node_modules/typescript/lib/typescript.js:16631:30)
    at Object.forEachChild (/Users/pschuster/Documents/Websites/fourplayers/node_modules/typescript/lib/typescript.js:16866:24)
    at checkNodeForDecorators (/Users/pschuster/Documents/Websites/fourplayers/node_modules/@angular-devkit/build-optimizer/src/transforms/scrub-file.js:44:31)
    at visitNode (/Users/pschuster/Documents/Websites/fourplayers/node_modules/typescript/lib/typescript.js:16622:24)
    at Object.forEachChild (/Users/pschuster/Documents/Websites/fourplayers/node_modules/typescript/lib/typescript.js:16864:21)
    at checkNodeForDecorators (/Users/pschuster/Documents/Websites/fourplayers/node_modules/@angular-devkit/build-optimizer/src/transforms/scrub-file.js:44:31)
    at visitNodes (/Users/pschuster/Documents/Websites/fourplayers/node_modules/typescript/lib/typescript.js:16631:30)
    at Object.forEachChild (/Users/pschuster/Documents/Websites/fourplayers/node_modules/typescript/lib/typescript.js:16859:24)
    at transformer (/Users/pschuster/Documents/Websites/fourplayers/node_modules/@angular-devkit/build-optimizer/src/transforms/scrub-file.js:39:16)
    at transformSourceFileOrBundle (/Users/pschuster/Documents/Websites/fourplayers/node_modules/typescript/lib/typescript.js:67384:57)
    at transformation (/Users/pschuster/Documents/Websites/fourplayers/node_modules/typescript/lib/typescript.js:83586:24)
```

**Comment by sglvladi** [comment]
Having the same issue here. Any help would be much appreciated. 

Thanks

**Comment by y4m4to** [comment]
I'm having the same issue using version 1.0.3 with Angular 8.3.1.

```
ERROR in /Users/yamato.aizawa/hrmony-prf/web/node_modules/@ctrl/ngx-emoji-mart/fesm2015/ctrl-ngx-emoji-mart.js
Module build failed (from /Users/yamato.aizawa/hrmony-prf/web/node_modules/@angular-devkit/build-optimizer/src/build-optimizer/webpack-loader.js):
TypeError: Cannot read property 'kind' of undefined
    at isAngularDecoratorMetadataExpression (/Users/yamato.aizawa/hrmony-prf/web/node_modules/@angular-devkit/build-optimizer/src/transforms/scrub-file.js:230:35)
    at checkNodeForDecorators (/Users/yamato.aizawa/hrmony-prf/web/node_modules/@angular-devkit/build-optimizer/src/transforms/scrub-file.js:53:21)
    at visitNodes (/Users/yamato.aizawa/hrmony-prf/web/node_modules/typescript/lib/typescript.js:16631:30)
    at Object.forEachChild (/Users/yamato.aizawa/hrmony-prf/web/node_modules/typescript/lib/typescript.js:16859:24)
    at transformer (/Users/yamato.aizawa/hrmony-prf/web/node_modules/@angular-devkit/build-optimizer/src/transforms/scrub-file.js:39:16)
    at transformSourceFileOrBundle (/Users/yamato.aizawa/hrmony-prf/web/node_modules/typescript/lib/typescript.js:67384:57)
    at transformation (/Users/yamato.aizawa/hrmony-prf/web/node_modules/typescript/lib/typescript.js:83586:24)
    at transformRoot (/Users/yamato.aizawa/hrmony-prf/web/node_modules/typescript/lib/typescript.js:83606:82)
    at Object.map (/Users/yamato.aizawa/hrmony-prf/web/node_modules/typescript/lib/typescript.js:499:29)
    at Object.transformNodes (/Users/yamato.aizawa/hrmony-prf/web/node_modules/typescript/lib/typescript.js:83593:30)
    at emitJsFileOrBundle (/Users/yamato.aizawa/hrmony-prf/web/node_modules/typescript/lib/typescript.js:84089:32)
    at emitSourceFileOrBundle (/Users/yamato.aizawa/hrmony-prf/web/node_modules/typescript/lib/typescript.js:84043:13)
    at forEachEmittedFile (/Users/yamato.aizawa/hrmony-prf/web/node_modules/typescript/lib/typescript.js:83832:34)
    at Object.emitFiles (/Users/yamato.aizawa/hrmony-prf/web/node_modules/typescript/lib/typescript.js:84026:9)
    at emitWorker (/Users/yamato.aizawa/hrmony-prf/web/node_modules/typescript/lib/typescript.js:89951:33)
    at /Users/yamato.aizawa/hrmony-prf/web/node_modules/typescript/lib/typescript.js:89912:66
    at runWithCancellationToken (/Users/yamato.aizawa/hrmony-prf/web/node_modules/typescript/lib/typescript.js:90002:24)
    at Object.emit (/Users/yamato.aizawa/hrmony-prf/web/node_modules/typescript/lib/typescript.js:89912:20)
    at Object.transformJavascript (/Users/yamato.aizawa/hrmony-prf/web/node_modules/@angular-devkit/build-optimizer/src/helpers/transform-javascript.js:111:13)
    at Object.buildOptimizer (/Users/yamato.aizawa/hrmony-prf/web/node_modules/@angular-devkit/build-optimizer/src/build-optimizer/build-optimizer.js:119:35)
    at Object.buildOptimizerLoader (/Users/yamato.aizawa/hrmony-prf/web/node_modules/@angular-devkit/build-optimizer/src/build-optimizer/webpack-loader.js:40:40)
```

But it worked by using version 1.0.1 🤔

**Comment by karptonite** [comment]
I just ran into the same issue with `ng build` not using universal, just normal AOT.

**Comment by scttcper** [comment]
some weird bug i think with ng-packagr. I think the latest version getting pushed soon should fix it or stay on v1.0.1 for now.

**Comment by karptonite** [comment]
Yeah, @scttcper  I actually had a post with "this seems like a problem with ng-packagr" in the text when you closed this--thanks for the quick action!  It looks like you rolled back to ng-packagr 5.5.1 from 5.7.0 in your 1.0.2 release. Is there a problem with ng-packagr 5.7.0? It looks like ng-packagr is leaving behind Angular 8, and I'd love a release of ngx-emoji-mart for angular 8 that builds and incorporates the  fix in `getWidth()` from 1.0.2, that affects us.
